controllers/snapshot/snapshot_adapter_test.go

```diff
@@ -700,6 +707,58 @@ var _ = Describe("Snapshot Adapter", Ordered, func() {
 			Expect(buf.String()).Should(ContainSubstring(expectedLogEntry))
 		})
 
+		It("no action when snapshot environmentBinding exists", func() {
```

Comment:
This should be rewritten to proper When statement with `BeaforeEach` and `AfterEach` to have working test cleanup.

Reply:
Updated.